airflow/config_templates/config.yml

```
@@ -2041,6 +2041,13 @@ celery:
       sensitive: true
       example: "db+postgresql://postgres:airflow@postgres/airflow"
       default: ~
+    result_backend_engine_options:
+      description: |
+        Optional options dictionary to pass to the Celery result backend SQLAlchemy engine.
```

"Optional options" is a mouthful, what about: 
```suggestion
        Optional configuration dictionary to pass to the Celery result backend SQLAlchemy engine.
```